.dockerignore

```diff
@@ -1,5 +1,6 @@
 *
-!.env
+.env*
+.next
```

Comment:
A bit superfluous as the first line of .dockerignore excludes everything `*` and the rest of the lines just say what is not excluded.